src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -30,6 +30,7 @@ internal abstract partial class AbstractAddImportFeatureService<TSimpleNameSynta
         protected abstract bool CanAddImportForMethod(string diagnosticId, ISyntaxFactsService syntaxFacts, SyntaxNode node, out TSimpleNameSyntax nameNode);
         protected abstract bool CanAddImportForNamespace(string diagnosticId, SyntaxNode node, out TSimpleNameSyntax nameNode);
         protected abstract bool CanAddImportForDeconstruct(string diagnosticId, SyntaxNode node);
+        protected abstract bool CanAddImportForGetAwaiter(string diagnosticId, SyntaxNode node, ISyntaxFactsService syntaxFactsService);
```

Comment:
super nitpicky.  but the ordering here vs CanAddImportForMethod annoys me.  Up to you if you wantt to fix it :)

Reply:
Would be nice for consistency. I'll make the change in a couple of hours when I'm free :)